src/ui/watchlist/searchById.ui.tsx

```diff
@@ -12,12 +12,12 @@ export default function SearchByIdUI({ land, metaverse }: SearchByIdUIProps) {
 
   const [tokenId, setTokenId] = useState<string>('');
   return (
-    <div className="flex flex-col">
-      <div className="w-full mb-4">
+    <div className="flex">
+      <div className="mr-6">
         <input
```

Comment:
remove (hide) arrows from the whole number inputs
![image](https://github.com/metagamehub/MGH-APP-V2.0/assets/106346968/e0a094ee-74b1-41be-a345-c21187b2b4c1)
https://www.w3schools.com/howto/howto_css_hide_arrow_number.asp